javascript/chronometer.js

```diff
@@ -1,26 +1,35 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    setInterval(
+      () => this.currentTime++, 1000);
```

Comment:
Great stuff implementing the setInterval method!
I would suggest two things:
- implement the call of the callback (conditionally) as mentioned in the iteration
- declare the function (the first argument of the setInterval method) outside of setInterval. Then, use it as the first argument. This makes the setInterval more readable.